### ISSUE TITLE
Remove taxize package from relevant functions

### ISSUE THREAD

**Issue by j-harbin** [dfo-mar-mpas/MarConsNetData#17]
As Remi pointed out the taxize package was removed from CRAN. See will instead explore the option of using the following option https://docs.ropensci.org/worrms/

**Comment by claudenozeres** [comment]
Note: was added (back?) on 2025-02-07: https://cran.r-project.org/web/packages/taxize/index.html

**Comment by remi-daigle** [comment]
Yep, it looks like it was added back, but we figured out how to get the same functionality from the worrms package (which we use for other purposes anyway), so we don't _need_ the taxize package anymore and it'll reduce our dependencies.

Thanks for the heads up!
